src/components/Sidebar.jsx

```diff
@@ -0,0 +1,196 @@
+})(({ theme, open }) => ({
+  width: drawerWidth,
+  flexShrink: 0,
+  whiteSpace: "nowrap",
+  boxSizing: "border-box",
+  ...(open && {
+    ...openedMixin(theme),
+    "& .MuiDrawer-paper": openedMixin(theme)
+  }),
+  ...(!open && {
+    ...closedMixin(theme),
+    "& .MuiDrawer-paper": closedMixin(theme)
+  })
+}));
+
+export default function MiniDrawer() {
+  const theme = useTheme();
+  const [open, setOpen] = React.useState(false);
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+  const navigate = useNavigate();
+
+  const itemList = [ 
+    {text: "Dashboard",
+    icon: <DashboardIcon /> ,
+    route: "dashboard"},
+    
+    {text: "Transactions",
+    icon: <CachedIcon />,
+    route: "transactions" },
+    {text: "Users",
+    icon: <PeopleAltIcon />, 
+    route: "users"},
+    {text: "Payouts",
+    icon: <ReceiptIcon />,
+    route: "payouts" }
+  ]
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <CssBaseline />
+      <AppBar position="fixed" open={open}>
+        <Toolbar >
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            onClick={handleDrawerOpen}
+            edge="start"
+            sx={{
+              marginRight: 5,
+              ...(open && { display: "none" })
+            }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap component="div" flexGrow={1}>
+            Lorem ipsum
+          </Typography>
+          <IconButton
+          position="right"
+            color="inherit"
+            // onClick={};
+            edge="end"
+          >
+            <PersonIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+      <Drawer variant="permanent" open={open}>
+        <DrawerHeader>
+          <IconButton onClick={handleDrawerClose}>
+            {theme.direction === "rtl" ? (
+              <ChevronRightIcon />
+            ) : (
+              <ChevronLeftIcon />
+            )}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List>
+          {itemList.map((item, index) => {
+            const {text, icon, route} = item;
+              return (
+              <ListItemButton
+                key={text}
+                onClick={() => navigate(route)}
+                sx={{
+                  minHeight: 48,
+                  justifyContent: open ? "initial" : "center",
+                  px: 2.5
+                }}
+              >
+                <ListItemIcon
+                  sx={{
+                    minWidth: 0,
+                    mr: open ? 3 : "auto",
+                    justifyContent: "center"
+                  }}
+                >
+                  {icon}
+                </ListItemIcon>
+                <ListItemText primary={text} sx={{ opacity: open ? 1 : 0 }} />
+              </ListItemButton>
+            )
+                })}
```

Comment:
please use prettier or another code formatter to properly indent components

Reply:
use this code formatter to format all files as they don't have proper indentation

Reply:
used prettier formatter for indentation